api/membership.go

```diff
@@ -81,7 +81,7 @@ func ApplyForMembershipHandler(app *App) func(c echo.Context) error {
 			txErr := app.Rollback(tx, "Membership application failed", c, logger, err)
 			if txErr != nil {
 				log.E(logger, "Could not rollback transaction", func(cm log.CM) {
-					cm.Write(zap.Error(txErr))
+					cm.Write(zap.Error(err))
```

Comment:
I don't think this is a good solution. If a rollback is failing, I want to know the reason it is failing, not the reason that made the code try to roll back in the first place.

I think that it would be better to have a new error log to log the `err` instead.